common/djangoapps/student/views.py

```diff
@@ -46,6 +46,8 @@
 
 from edxmako.shortcuts import render_to_response, render_to_string
 
+from enterprise.tpa_pipeline import active_provider_requests_data_sharing, active_provider_enforces_data_sharing
+from enterprise.models import EnterpriseCustomer
```

Comment:
`enterprise` is an optional app, so you can't just import from it in shared code without a `try`/`except`